_posts/2019-09-03-how-to-deploy-production-grade-kubernetes-cluster-aws.adoc

```diff
@@ -1459,19 +1502,21 @@ data "template_file" "user_data" {
 }
 ----
 
+# TODO: Shouldn't we mention to make the user-data.sh executable with chmod?
```

Comment:
I think it gets uploaded via AWS APIs as text, not a file, so prob doesn't matter